react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx

```diff
@@ -1,22 +1,36 @@
-import React, { Fragment, useState, FC } from 'react'
-import { Button, Modal } from 'vtex.styleguide'
-import ProductPrice from 'vtex.store-components/ProductPrice'
+import React, { FC, useState } from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
 import { useDevice } from 'vtex.device-detector'
-import { FormattedMessage } from 'react-intl'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+import { Button, Modal } from 'vtex.styleguide'
 
+import { ProductAssemblyGroupContextProvider } from '../ProductAssemblyContext/Group'
 import { useProductAssemblyItem } from '../ProductAssemblyContext/Item'
-import ProductAssemblyOptionsGroup from './ProductAssemblyOptionsGroup'
+import StopPropagation from '../StopPropagation'
 import { imageUrlForSize } from './ProductAssemblyOptionItemImage'
-import { ProductAssemblyGroupContextProvider } from '../ProductAssemblyContext/Group'
+import ProductAssemblyOptionsGroup from './ProductAssemblyOptionsGroup'
 import { withItem } from './withItem'
-import StopPropagation from '../StopPropagation'
+
+const CSS_HANDLES = [
+  'modalViewProductContainer',
+  'modalViewProductImage',
+  'modalViewProductName',
+  'modalViewProductInfos',
+  'modalViewDoneButton',
+  'modalViewTitle',
+  'productAssemblyOptionItemCustomize',
+  'productAssemblyOptionItemCustomize__label',
```

Comment:
Can you rename it to camelCase?

```suggestion
  'productAssemblyOptionItemCustomizeLabel',
```